### PR TITLE
Enhancement/improve comments

### PR DESCRIPTION
**Pull request by ohommos**
- removed commented code
- removed unnecessary variables
- removed TODOs
- added extra comments for methods longer than 10 LOC
- Organized imports; removed un-necessaries.

### CODE REVIEW

File: core/src/com/group66/game/cannon/BallsStatic.java [hunk 1]
```diff
@@ -255,7 +255,6 @@ public void moveRowDown() {
      */
     public boolean hitsBottom() {
         for (Ball b : ballStaticList) {
-            // TODO fix the != check
```
Comment (ALobker) [inline core/src/com/group66/game/cannon/BallsStatic.java]: @everyone Has this been resolved?

**Review by ALobker — APPROVED** [review]
The code looks wonderful. Good job.

File: core/src/com/group66/game/cannon/BallsStatic.java (hunk 1 above)
Comment (iiKoe) [inline core/src/com/group66/game/cannon/BallsStatic.java]: Yea, I was thinking about that too, I think the check can go now, but I'm not shure.
If we remove the check (which was for the cannon ball up next) we need to test real good that it still works

**Review by iiKoe — APPROVED** [review]
Looks good, please do consider Antony's comment

**Comment by 11maxed11** [comment]
Will solve the conflicts and then merge.
@iiKoe @ALobker What do you recommend doing? the TODO comment was removed as it won't be solved now anyway and the code has to be submitted clean. I suggest not changing the code now. works this way == this way is perfect..

**Comment by iiKoe** [comment]
Yea, the check is probably not needed, but removing it now is too scary. Leave it

**Pull request merged**